frontend/src/components/annotation/audio_annotation/audio_annotation_core.vue

```diff
@@ -254,90 +245,31 @@ export default {
         this.$emit("request_file_change", direction, file);
       }
     },
-    save: async function(){
-      this.has_changed = false
-      this.save_loading = true
-      let url;
-      if (this.task && this.task.id) {
-        url = `/api/v1/task/${this.task.id}/annotation/update`;
-      } else {
-        url = `/api/project/${this.project_string_id}/file/${this.file.id}/annotation/update`
-      }
-
-      const res = await postInstanceList(url, this.instance_list.get_all())
-      const { added_instances } = res
-      this.instance_list.get_all().map(instance => {
-        const instance_uuid = instance.creation_ref_id
-        const updated_instance = added_instances.find(added_instance => added_instance.creation_ref_id === instance_uuid)
-        if (updated_instance) {
-            instance.id = updated_instance.id
-        }
-      })
-      this.save_loading = false
-    },
-    trigger_task_change: async function (direction, assign_to_user = false) {
```

Comment:
`trigger_task_change`, and `on_task_annotation_complete_and_save` since this actions are handeled by the ancestor component. Also `text_annotation_core` didn't have them.

Reply:
We can test this by going to task UI context and completing the task.